### PR TITLE
Auth signout

### PR DESCRIPTION
**Pull request by jeffrey-dot-li**

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React, { ReactNode, ReactComponentElement, ReactElement,  } from 'react';
```
Comment (Eshchock1) [inline src/components/Card.tsx]: I don't see any changes in this file. Have any changes been made?

File: src/components/CardList.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```
Comment (Eshchock1) [inline src/components/CardList.tsx:1]: I don't see any changes in this file either.

**Comment by Eshchock1** [comment]
Sign out and sign in works really well. The menu is very clean and smooth, really solid job, but you should consider making the option to access the menu more clear.

File: src/components/Header.tsx [hunk 3]
```diff
@@ -1,39 +1,130 @@
...
+      <Block flex={false} row style={{paddingVertical: 15}}>
+      {
+      backNavigationCB ? (
+          <TouchableOpacity activeOpacity={0.8} onPress={backNavigationCB} style={{...styles.back}}> 
+              <IoniconsIcon name="ios-arrow-round-back" size={45} color="white"/>            
+          </TouchableOpacity>
+      ) : null
+      }
+      <Block center flex={1} >
+        <Text h3 white style={{fontSize:21}} numberOfLines={1}>
```
Comment (Eshchock1) [inline src/components/Header.tsx]: The page name is pushed to the left by the user profile picture by 35, so make sure to add marginLeft:35 to the text property so the page name is centered.

File: src/components/Header.tsx [hunk 4]
```diff
@@ -1,39 +1,130 @@
...
+  const hideMenu = () => {
+    _menu.hide();
+  };
+ 
+  const showMenu = () => {
+    _menu.show();
+  };
+
+  const signOut = async () => {
+    dispatch<any>(actions.user.logoutUser()) // huh this is dumb
```
Comment (Eshchock1) [inline src/components/Header.tsx:45]: add confirmation popup before signing out

File: src/assets/styles/styles.ts [hunk 5]
```diff
@@ -1,19 +1,32 @@
 import {    StyleSheet } from 'react-native'
 import * as theme from '../../constants/theme';
+import { stackOffsetNone } from 'd3-shape';
```
Comment (seendsouza) [inline src/assets/styles/styles.ts:3]: where did this come from

**Review by seendsouza — APPROVED** [review]

**Pull request merged**